skbio/stats/ordination/_ordination_results.py

```diff
@@ -399,3 +399,44 @@ def _format_attribute(self, attr, attr_label, formatter):
         else:
             formatted_attr = formatter(attr)
         return "\t%s: %s" % (attr_label, formatted_attr)
+
+    def rename(self, mapper, axis='samples', strict=True):
+        """
+        Renames ids in OrdinationResults.
+
+        Parameters
+        ----------
+        mapper : dict or function
+            A dictionary or function that maps current ids to new ids.
+        axis : samples or features
+            Specifies which id type, either 'samples' or 'features', to rename.
+        strict: bool, optional
+           If True, then every id in the OrdinationResults dataframe must be
+           included in rename_dict and renamed.
+           If False, then only the specified indices will be renamed.
+
+        Raises
+        ______
+        ValueError
+            If strict is True and rename_dict does not contain all of the same
+            ids as OrdinationResults.
+
+        """
+        if axis == 'samples':
+            if (strict and isinstance(mapper, dict) and
+                    set(mapper) != set(self.samples.index)):
+                raise ValueError("The ids in mapper are different from the \
+                                 ids in self.samples.")
+            self.samples = self.samples.rename(index=mapper)
+        elif axis == 'features':
+            if self.features is None:
+                raise ValueError("`features` were not provided on \
+                                 construction of this object")
+            elif (strict and isinstance(mapper, dict) and
+                    set(mapper) != set(self.features.index)):
+                raise ValueError("The ids in mapper are different \
+                                  from the ids in self.features.")
+            self.features = self.features.rename(index=mapper)
+        else:
+            raise ValueError("axis must be either 'samples' or \
```

Comment:
`axis` should be capitalized because it is a sentence.

Reply:
Done!